scripts/config.ts

```diff
@@ -489,4 +531,4 @@ export const writeL3ChainConfigCommand = {
     handler: (argv: any) => {
         writeL3ChainConfig(argv)
     }
-}
+}
```

Comment:
Unnecessary changes

Reply:
I agree that this is unnecessary, but I don't know what I would do to fix this part of the commit log. I had added and removed a function after this brace. However, I didn't actually modify it at all, so I'm not exactly sure why git is registering it as a changed line.

Reply:
Are you using some formatter extension to remove the trailing new line?